julearn/api.py

```diff
@@ -195,8 +199,11 @@ def run_cross_validation(  # noqa: C901
         logger.info(f"Setting random seed to {seed}")
         np.random.seed(seed)
 
+    if data is None:
```

Comment:
Maybe this needs change of the argument to be positional? It would still need updating the docstring to have correct type or a `Raises` section depending on the answer of the question.